api/golang/api_constructors/constructors.go

```diff
@@ -42,6 +43,7 @@ func NewKurtosisPackage(
 		Version:               version,
 		IconUrl:               iconURL,
 		RunCount:              runCount,
+		LocatorRoot:           locatorRoot,
```

Comment:
The locator root in the documentation is what is in kurtosis.yml (without the main.star).  Root typically refer to a directory.  Should we rename it to Locator instead?

Reply:
good suggestion @laurentluce Ill do it